client/Routes.js

```diff
@@ -0,0 +1,98 @@
+
+import React, { Component } from 'react'
+import {connect} from 'react-redux'
+import { withRouter, Route, Switch, Redirect } from 'react-router-dom'
+import { Login, Signup } from './components/AuthForm';
+import Home from './components/Home';
+import { me, fetchCart, fetchProducts, fetchUsers } from './store'
+import Cart from './components/Cart';
+import Products from './components/Products';
+import Product from'./components/Product';
+import About from './components/About';
+import Orders from './components/Orders';
+import Order from './components/Order';
```

Comment:
typically in production apps (like big apps), you want to "order" the imports just to organize it more

the order goes as follows- 
```js
// all of your library imports
import React from 'react';
import { connect } from 'react-dom';
import { Route } from 'react-router-dom';

// all of your components
import Nav from './Nav';
import Product from './Product';

// all of your store stuff
import { loadUsers, loadCart } from './store';
```